spec/lib/formatter/csv/annotation_for_csv_spec.rb

```diff
@@ -138,67 +138,95 @@
       end
 
       context "with a combo task" do
-        let(:combo_workflow) { build(:workflow, :complex_task) }
```

Comment:
complex tasks have a mix on combo and others [since this](https://github.com/zooniverse/Panoptes/commit/2c2a49ccdaffd89908da49c6281d56253a1501a3), and the existing specs require that specific workflow configuration. 

I agree that the wording is off, i'll update the let variable name to make it clearer.